### ISSUE TITLE
Prompt stuck in Linux VT console

### ISSUE THREAD

**Issue by yegorius** [franciscolourenco/done#42]
Prompt hangs if fish is being run from Linux VT console.
This happens because `test -n __done_get_focused_window_id` always evaluates to `true`, regardless of DISPLAY variable being set.

**Comment by bfred-it** [comment]
It's happening for me too:

```
macOS 10.14.3
fish, version 3.0.1
fisher version 3.2.7
```

I have to press `ctrl+c` to end it or click the notification, which makes this appear:

```
2019-03-29 10:55:48.460 terminal-notifier[29736:36262668] @CONTENTCLICKED
```

This actually seems to be a bug with `terminal-notifier` because after running `brew uninstall terminal-notifier` this bug no longer shows up.

**Comment by yegorius** [comment]
I know my fork is only relevant for Linux, but this is how I solved the problem in my case.
https://github.com/yegorius/done

**Comment by franciscolourenco** [comment]
@fregante These seem to be separate issues then. @fregante would you mind creating a new ticket for the terminal-notifier problem so I can have a look into it separately?

@yegorius Thanks for the branch I will have a look!

**Comment by franciscolourenco** [comment]
@fregante have you tried updating terminal notifier? I have version 2.0 installed and cannot reproduce

**Comment by franciscolourenco** [comment]
@yegorius which distro are you using?

**Comment by franciscolourenco** [comment]
@yegorius I've pushed a fix. Could you please confirm that it works by uninstalling your fork and installing with 
```
fisher addfranciscolourenco/done:fix-linux-vt
```

Thanks!

**Comment by yegorius** [comment]
It works ok, I am not experiencing the bug any more.

**Comment by franciscolourenco** [comment]
Released in 1.8.1
```
fisher remove franciscolourenco/done:fix-linux-vt
fisher add franciscolourenco/done
```
